HW1/P2/P2a.py

```diff
@@ -1,3 +1,30 @@
 from P2 import *
+import numpy as np
+import matplotlib.mlab as mlab
+import matplotlib.pyplot as plt
 
-# Your code here
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext(appName="P2")
+
+
+rdd1 = np.arange(2000)
+rdd2 = np.arange(2000)
+
+rdd1 = sc.parallelize(rdd1, 10)
+rdd2 = sc.parallelize(rdd2, 10)
+
+# Create the pixels rdd
+pixels = rdd1.cartesian(rdd2)
```

Comment:
In order to avoid all of the above duplication, it might have been cleaner to simply make a single rdd and then cartesian it with itself (ie `rdd = sc.parallelize(np.arange(2000), 10)` and then `rdd.cartesian(rdd)`)